CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added value_and_grad function to the autograd plugin, importable via `from tidy3d.plugins.autograd import value_and_grad`. Supports differentiating functions with auxiliary data (`value_and_grad(f, has_aux=True)`).
 
+### Fixed
+- `DataArray` interpolation failure due to incorrect ordering of coordinates.
```

Comment:
maybe add something mentioning "containing autograd tracers" so people dont think this was broken more generally before or broken in xarray.

Reply:
Oops I thought I added that.